deployment/copilot/createReplayer.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+
+# Wish list
+# * Allow adding additional security group ids for the Replayer service
```

Comment:
It could be fruitful to write this up as a "first-task" and link to the issue for any enterprising users.

Reply:
Good thought, I have added a link to an issue here with label 'good-first-issue'